decoder/src/lib.rs

```diff
@@ -1608,6 +1621,31 @@ mod tests {
         );
     }
 
+    #[test]
+    fn bitfields_u128() {
```

Comment:
I was a bit surprised about this test because expected that `{0=119..124:b}` is encoded as 2 bytes instead of as 16 bytes. After checking locally, this test also works if I remove the last 14 bytes. It would be nice to make `decode_and_expect` check that the input data was fully consumed to ensure we are not passing extra bytes that will be ignored.